### ISSUE TITLE
Add a version tag on AssetLibs, to show the godot versions that the asset have support.

### ISSUE THREAD

**Issue by edson-dev** [godotengine/godot-proposals#10507]
### Describe the project you are working on

on working on a migration of godot 3 to 4

### Describe the problem or limitation you are having in your project

i dont have information on the tab of asset libs of the minimal support of the libs.

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

add a tag on the asset description with support versions, set by the creator so we have an insight about version support description

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

when you click on the asset or in the list assets have a support-[3.0,4.2,4.3] tag that can be set at the plugin.cfg by the creators. Its a start to map support for plugins and versions

### If this enhancement will not be used often, can it be worked around with a few lines of script?

i dont see any rason why dont let creator map supported versions. It can help get info and help.

### Is there a reason why this should be core and not an add-on in the asset library?

need change the package descriptions and base config parameters.

**Comment by Calinou** [comment]
There is already a per-asset version property you can see on the web frontend: https://godotengine.org/asset-library/asset

It defines the minimum supported Godot version, with newer minor releases being considered as supported (they will be able to see the asset in the editor). This matches [Godot's release policy](https://docs.godotengine.org/en/latest/about/release_policy.html) where backwards compatibility is preserved as much as possible within a given major release.

As the Submit Asset page says:

![image](https://github.com/user-attachments/assets/79de1c9a-d0e4-4a6c-bfb8-32d69582c3c3)

**Comment by timothyqiu** [comment]
> It defines the minimum supported Godot version, with newer minor releases being considered as supported (they will be able to see the asset in the editor).

Can an asset with the same name (probably also the same author) and a higher minimum version override an older version?

For example, Asset Foo v1.0 supports Godot 4.1. But some changes in 4.3 break Asset Foo's code. So it has to release a v2.0 that supports Godot 4.3. Ideally, this should make v1.0 only support `[4.1, 4.3)`, and make v2.0 support `[4.3, 5.0)`

**Comment by Calinou** [comment]
> Can an asset with the same name (probably also the same author) and a higher minimum version override an older version?

No, as the current backend can only define a single literal version string, not a [semver string](https://semver.org/) (which would allow for this to be defined).
